pkg/gofr/handler.go

```diff
@@ -126,13 +128,14 @@ func (h handler) setContextTimeout(timeout string) int {
 	return reqTimeout
 }
 
-func panicRecoveryHandler(log logging.Logger, panicked chan struct{}) {
-	re := recover()
-	if re != nil {
-		close(panicked)
-		log.Error(panicLog{
-			Error:      fmt.Sprint(re),
-			StackTrace: string(debug.Stack()),
-		})
+func panicRecoveryHandler(re any, log logging.Logger, panicked chan struct{}) {
```

Comment:
@ccoVeille why not call the `recover()` in panicRecoveryHandler instead of passing re!

Reply:
Honestly, I wonder too

That's the only way I found to cimply with the revive.defer linter.

Adding a //nolint to the function was not enough.

Maybe there is something I missed. Someone might find a solution, but for now. The recover call is clear now.

Reply:
Yeah, not sure why the linter is doing it, I think we can change it later if we can find a solution to this.